usr/lib/systemd/system/rbd-target-api.service

```diff
@@ -2,9 +2,8 @@
 Description=Ceph iscsi target configuration API
 
 Requires=sys-kernel-config.mount
-After=sys-kernel-config.mount network-online.target rbd-target-gw.service
-BindsTo=rbd-target-gw.service
-Wants=network-online.target
+After=sys-kernel-config.mount network-online.target
```

Comment:
Should ```tcmu-runner``` move to this service as well?

Reply:
Yes, and I missed the TimeoutStopSec part too. Fixed and repushed.